.github/workflows/camunda-platform-release.yml

```diff
@@ -336,7 +326,7 @@ jobs:
           name: ${{ inputs.releaseVersion }}
           artifacts: "release-artifacts/*"
           artifactErrorsFailBuild: true
-          draft: ${{ steps.gen-changelog.outputs.isDraftRelease }}
+          draft: false
```

Comment:
why is this changed? this means any release will be created as draft.

Reply:
according to https://github.com/camunda/camunda/issues/32376#issuecomment-2940124969 they need to be non-draft always.